opam-monorepo.opam.locked

```diff
@@ -93,19 +93,19 @@ pin-depends: [
     "https://github.com/dune-universe/astring/archive/v0.8.5+dune.tar.gz"
   ]
   [
-    "base.v0.14.1" "https://github.com/janestreet/base/archive/v0.14.1.tar.gz"
+    "base.v0.14.3" "https://github.com/janestreet/base/archive/v0.14.3.tar.gz"
   ]
   [
-    "bigarray-compat.1.0.0"
-    "https://github.com/mirage/bigarray-compat/archive/v1.0.0.tar.gz"
+    "bigarray-compat.1.1.0"
+    "https://github.com/mirage/bigarray-compat/releases/download/v1.1.0/bigarray-compat-1.1.0.tbz"
   ]
   [
     "bigstringaf.0.8.0"
     "https://github.com/inhabitedtype/bigstringaf/archive/0.8.0.tar.gz"
   ]
   [
-    "bos.0.2.0+dune"
-    "https://github.com/dune-universe/bos/archive/v0.2.0+dune.tar.gz"
+    "bos.0.2.1+dune"
+    "git+https://github.com/dune-universe/bos.git#e8fa18765d4538bc5d6cc1501891d0cde3d1a797"
```

Comment:
Shouldn't we better have a tarball here?

Reply:
good point, I'll go and update the port!

Reply:
I'm a bit confused, because I just checked the port and we have a regular tarball of 0.2.1+dune in our repo, which is also referenced by opam-overlays correctly. Maybe you have a pin that is interfering?